### ISSUE TITLE
Sourcecred Contributor Wall not Optional

### ISSUE THREAD

**Issue by nothingismagick** [sfosc/sfosc#136]
I love the sourcecred wall - but there is a concern that I have, and that is that not everyone who we are presenting there has opted into it, and their appearance might be misconstrued as their explicit acceptance of this project and its rather opinionated statements.

I am not sure what the best way around this is, but I can see two options:

1. Rephrase it, e.g.:
```
## Participants
Many thanks to the people who have participated in the discussions.
```

2. Use an opt-in approach, where people have to explicitly request / permit it. I would like to assume that "members" have given their permission, but this is also not the case.

**Comment by Beanow** [comment]
I hope that 1) would be an option and we can find a correct wording for it. However I feel like there is always a risk of making it look like an endorsement / association. So if we do choose 1) there may still be a need to opt-out.
2) would be the cautious route. It's a fair bit of overhead probably and may be a missed chance for attribution? So I feel like my preference would be 1). Making it similar to DCMA, providing an opt-out.

**Comment by nothingismagick** [comment]
Yeah @decentralion - I tend to agree with your assessment, mostly because people with low interaction just don't feel like their activity is necessarily a contribution. I had this issue once before where I tried mentioning LOTS of people in an issue and many of them responded that they didn't understand why they were being mentioned.

**Comment by Beanow** [comment]
On this issue and the part of finding better thresholds / interpreting cred scores I posted here https://discourse.sourcecred.io/t/interpreting-scores-widgets/211
